Resources/Textures/SS220/Objects/Fun/Lottery.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt",
+  "copyright": "Created by VladIsLove and discord glek3197 for SS220 ",
```

```suggestion
  "copyright": "Created by glek3197 (Discord) for SS220",
```
спрайт рисовал глек, копирайт - его

---

ой баляяяять